src/main/menus/app.ts

```diff
@@ -472,21 +472,27 @@ export function getAppMenu() {
         },
         {
           label: `Hide ${app.name}`,
-          accelerator: 'CommandOrControl+H',
+          visible: is.macos,
           role: 'hide'
         },
         {
           label: 'Hide Others',
-          accelerator: 'CommandOrControl+Shift+H',
+          visible: is.macos,
           role: 'hideOthers'
         },
         {
           label: 'Show All',
+          visible: is.macos,
           role: 'unhide'
         },
-        {
-          type: 'separator'
-        },
+        is.macos
+          ? {
+              type: 'separator'
+            }
+          : {
+              label: 'hidden separator',
+              visible: false
+            },
```

Comment:
This could be simplified like this:

```ts
        ...(is.macos
          ? ([
              {
                label: `Hide ${app.name}`,
                role: 'hide'
              },
              {
                label: 'Hide Others',
                role: 'hideOthers'
              },
              {
                label: 'Show All',
                role: 'unhide'
              },
              {
                type: 'separator'
              }
            ] as MenuItemConstructorOptions[])
          : []),
```

Reply:
I think you're missing the spread `...` operator. Look closely at my code example at the beginning.

Reply:
Ah, I see, thanks!